src/lib/services/txService.ts

```diff
@@ -161,3 +166,75 @@ export const useTxsCountByAddress = (
     }
   );
 };
+
+export const useTxs = (
+  offset: number,
+  pageSize: number
+): UseQueryResult<Transaction[]> => {
+  const { indexerGraphClient } = useCelatoneApp();
+
+  const queryFn = useCallback(
+    async () =>
+      indexerGraphClient
+        .request(getTxs, {
+          offset,
+          pageSize,
+        })
+        .then(({ transactions }) =>
+          transactions.map((transaction) => ({
+            hash: parseTxHash(transaction.hash),
+            messages: snakeToCamel(transaction.messages),
+            sender: transaction.account.address as Addr,
+            isSigner: false,
+            height: transaction.block.height,
+            created: parseDateOpt(transaction.block.timestamp),
```

Comment:
It is no longer optional. Change the value type to `Date` and get rid of the 'N/A' handling.